### PR TITLE
feat: Remove obsolete reconciliations inline for all clusters

### PR DESCRIPTION
**Pull request by adityabhatia**
- now also includes reconciliations from deleted clusters (changes RuntimeID)

### CODE REVIEW

File: pkg/model/clusterconfigentity.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ type ClusterConfigurationEntity struct {
 	ClusterVersion int64            `db:"notNull"` // Cluster entity primary key
 	KymaVersion    string           `db:"notNull"`
 	KymaProfile    string           `db:""`
-	Components     []*keb.Component `db:"notNull,encrypt"`
+	Components     []*keb.Component `db:"notNull"`
```
Comment (adityabhatia) [inline pkg/model/clusterconfigentity.go:31]: remove

File: pkg/model/clusterconfigentity.go [hunk 2]
```diff
@@ -28,7 +28,7 @@ type ClusterConfigurationEntity struct {
 	ClusterVersion int64            `db:"notNull"` // Cluster entity primary key
 	KymaVersion    string           `db:"notNull"`
 	KymaProfile    string           `db:""`
-	Components     []*keb.Component `db:"notNull,encrypt"`
```
Comment (tobiscr) [inline pkg/model/clusterconfigentity.go:31]: field has to be encrypted actually

**Review by tobiscr — CHANGES_REQUESTED** [review]

File: pkg/model/clusterconfigentity.go (hunk 2 above)
Comment (adityabhatia) [reply pkg/model/clusterconfigentity.go:31]: sure, already added a comment above to clean it :)

File: pkg/scheduler/reconciliation/inmemory.go [hunk 3]
```diff
@@ -122,22 +122,28 @@ func (r *InMemoryReconciliationRepository) RemoveReconciliationByRuntimeID(runti
...
+			delete(r.operations, schedulingID)
+			delete(r.reconciliations, runtimeID)
+		}
 	}
 
 	return nil
 }
 
+func (r *InMemoryReconciliationRepository) GetRuntimeIDs() ([]string, error) {
+	return nil, nil
```
Comment (adityabhatia) [inline pkg/scheduler/reconciliation/inmemory.go:144]: implement..

**Review by khlifi411 — APPROVED** [review]

**Pull request merged**